lib/src/main/java/io/cloudquery/schema/Table.java

```diff
@@ -210,6 +210,10 @@ public Optional<Column> getColumn(String name) {
     return Optional.empty();
   }
 
+  public Optional<TableResolver> getResolver() {
```

Comment:
See https://medium.com/@johnpang/optionals-and-lombok-a414639dca1e

Reply:
Good advice - Optional shouldn't be a field, just the getter.